integration/fuse/src/main/java/alluxio/jnifuse/struct/FuseFileInfo.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.jnifuse.struct;
+
+import java.nio.ByteBuffer;
+
+public class FuseFileInfo extends Struct {
```

Comment:
- depending on the fuse version, can you duoble check this is consistent with https://github.com/libfuse/libfuse/blob/fuse_2_9_bugfix/include/fuse_common.h (if we target fuse 2.9)
- please include the link to the header file in the javadoc
- include in javadoc that this works with only Linux or MacOS (if that is the case)